# Aufgabe 8.1

Der skeptische Konsument gibt nicht auf und versucht weiterhin, den Weinhändler des Betrugs zu überführen. DerWeininhalt der 12 erworbenen Weinflaschen lauten:

```
71, 69, 67, 68, 73, 72, 71, 71, 68, 72, 69, 72 (in Zentiliter)
```

**a) Nun zweifeln wir daran, ob die Datenwirklich gut durch eine Normalverteilung beschriebenwerden können (diese Annahme haben wir sowohl beim z- als auch beim t-Test gemacht). Wenn die Normalverteilungsannahme nicht gemacht werden kann, können wir den Vorzeichen-Test durchführen. Führen Sie also den Vorzeichen-Test auf dem 5%-Signifikanzniveau durch. Wie lautet nun das Ergebnis?**

In [21]:
import scipy.stats as st
import numpy as np

# H_0: mean = 70
x = np.array([71, 69, 67, 68, 73, 72, 71, 71, 68, 72, 69, 72])

# Positive Vorzeichen zählen
n_positives = x[ x - 70 > 0].size

p = st.binom.cdf(k=n_positives, n=x.size, p=0.5)
print('p-Wert = {:.3f}'.format(p))

p-Wert = 0.806



Mit einem Signifikanzniveau von $5\%$ kann die Nullhypothese bei einem p-Wert von $0.806$ nicht verworfen werden.

Der Weinhändler betrügt wahrscheinlich nicht.

**b) Wie lautet das Ergebnis mit dem Wilcoxon-Test?**

In [26]:
import scipy.stats as st

diff = x - 70
p = st.wilcoxon(x=diff, correction=True).pvalue
print('p={:.3f}'.format(p))

p=0.690


# Aufgabe 8.2

Untenstehend finden Sie mehrere Beispiele für Vergleiche von 2 Stichproben. Beantworten Sie für jedes Beispiel kurz die folgenden Fragen:
    
- Handelt es sich um gepaarte oder um ungepaarte Stichproben? Begründen Sie!
- Ist der Test einseitig oder zweiseitig durchzuführen? Begründen Sie!
- Wie lautet die Nullhypothese in Worten?
- Wie lautet die Alternativhypothese in Worten?

**a) In einem Experiment sollte der Effekt von Zigarettenrauchen auf Blutplättchenanhäufungen untersucht werden. Dazu wurden 11 Probanden vor und nach dem Rauchen einer Zigarette Blutproben entnommen, und es wurde gemessen, wie stark sich die Blutplättchen anhäuften. Es interessiert, ob sich Blutplättchen durch das Rauchen vermehrt anhäufen.**

- *Gepaart*: Es handelt sich um die selben Probanden. Jeder Messung vor dem Rauchen kann also eindeutig eine Messung nach dem Rauchen zugewiesen werden.
- *Einseitig*: Es wird untersucht, ob sich die Blutplättchen anhäufen
- $H_0$: Rauchen hat keinen Einfluss auf die Anzahl Blutplättchen: $\mu_R = \mu_{NR}$
- $H_A$: Rauchen erhöht die Anzahl Blutplätthen: $\mu_R > \mu_{NR}$

**b) Die nächsten Daten sind aus einer Studie von Charles Darwin über die Fremd- und Selbstbefruchtung. 15 Paare von Setzlingen mit demselben Alter, je einer durch Selbst- und einer durch Fremdbefruchtung produziert, wurden gezüchtet. Beide Teile je eines Paares hatten nahezu gleiche Bedingungen. Das Ziel bestand darin zu sehen, ob die fremdbefruchteten Pflanzen mehr Lebenskraft besitzen als die selbstbefruchteten (d.h., ob sie grösser werden). Es wurden die Höhen
jeder Pflanze nach einer fixen Zeitspanne gemessen.**

- *Gepaart*: Jede Messung des selbstbefruchteten Setzlings kann einer Messung des entsprechenden fremdbefruchteten Setlings zugeordnet werden.
- *Einseitig*: Es sollte untersucht werden, ob die fremdbefruchteten Setzlinge grösser sind.
- $H_0$: Die Höhen unterscheiden sich nicht.
- $H_A$: Fremdbefruchtete Setlinge werden grösser als selbsbefruchtete.

**c) Beeinflusst der Kalziumgehalt in der Nahrung den systolischen Blutdruck? Zur Überprüfung dieser Frage wurde einer Versuchsgruppe von 10 Männern während 12 Wochen ein Kalziumzusatz verabreicht. Einer Kontrollgruppe von 11 Männern gab man ein Placebopräparat.**

* *Ungepaart*: Verschiedene Männer, unterschiedliche Grössen der Gruppen
* *Zweiseitig*: Es soll untersucht werden, ob Kalzium einen Einfluss hat (positiv oder negativ.
* $H_0$: Kalzium hat keinen Einfluss auf den Blutdruck.
* $H_A$: Kalzium hat einen Einfluss auf den Blutdruck.

**d) In einem Experiment wurde untersucht, ob Mäuse zwei Formen von Eisen (Fe2+ und Fe3+) unterschiedlich gut aufnehmen. Dazu wurden 36 Mäuse in zwei Gruppen zu je 18 unterteilt und die eine Gruppe mit Fe2+ und die andere mit Fe3+ „gefüttert”. Da das Eisen radioaktiv markiert war, konnte sowohl die Anfangskonzentration wie auch die Konzentration einige Zeit später gemessen werden. Daraus wurde für jede Maus der Anteil des aufgenommenen Eisens berechnet.**

* *Ungepaart*: Es kann nicht jeder Maus aus einer Gruppe eindeutig eine Maus aus der anderen Gruppe zugewiesen werden.
* *Zweiseitg*: Es sollt untersucht werden ob es einen Unterschied gibt.
* $H_0$: Es gibt keinen Unterschied.
* $H_A$:  Es gibt einen Unterschied.

# Aufgabe 8.3

Zwei Tiefen-Messgeräte messen für die Tiefe einer Gesteins-Schicht an 9 verschiedenen Orten die folgenden Werte: Kennzahlen für die Differenz: $\overline{d}_n$ beträgt −5.78, die Standardabweichung $\sigma_D$ = 6.2.

```
Messgerät A: 120 265 157 187 219 288 156 205 163
Messgerät B: 127 281 160 185 220 298 167 203 171
Differenz d:  −7 −16  −3   2  −1 −10 −11   2  −8
```

Es wird vermutet, dass Gerät B systematisch grössere Werte misst. Bestätigen die Messwerte diese Vermutung oder ist eine zufällige Schwankung als Erklärung plausibel?

**a) Handelt es sich um verbundene (gepaarte) oder um unabhängige Stichproben?**

Gepaart: Jede Messung mit Gerät A kann einer Messung mit Gerät B *am gleichen Ort* zugewiesen werden.

**b) Führen Sie einen t-Test auf dem Niveau a = 0.05 durch. Formulieren Sie explizit: Modellannahmen,Nullhypothese, Alternative, Teststatistik, Verwerfungsbereich und Testergebnis.**

- *Modell*: $D_i iid \sim \mathcal{N}(\mu, \sigma^2)$, $\sigma$ wird durch $\hat{\sigma}$ geschätzt
- $H_0$: $\mu_D = \mu_0 = 0$
- $H_A$: $\mu_D < \mu_0$
- *Teststatistik*: t-Verteilung mit Freiheitsgrad 8
- *Signifikanzniveau: $\alpha = 0.05$

In [30]:
from scipy.stats import t
import numpy as np

t.ppf(q=0.05, loc=0, scale=6.2/np.sqrt(9), df=8)

-3.8430659442138757

- *Verwerfungsbereich*: $K = (-\infty, -3.843]$
- *Testentscheid*: Der Wert $\overline{d}_n= −5.78$ ist im Verwerfungsbereich, die Nullhypothese wird also verworfen.

Gerät B misst signifikant grössere Werte als Gerät A.

**c) Sei Z die Zufallsvariable, die zählt, bei wie vielen der 9 Messungen Gerät A einen grösseren Wert misst, als Gerät B. Wie ist Z verteilt, wenn die Geräte bis auf Zufallsschwankungen das Gleiche messen?**

Z ist binomialveteilt.

# Aufgabe 8.4

In der folgenden Tabelle sind die Kieferlängen von 10 männlichen und 10 weiblichen Goldschakalen eingetragen:

Einige Kennzahlen:

* $\overline{x}_n = 113.4$, $\hat{\sigma}_x^2 = 13.82$
* $\overline{y}_n = 108.6$, $\hat{\sigma}_y = 5.16$.

```
männlich (x_i): 120 107 110 116 114 111 113 117 114 112
weiblich (y_j): 110 111 107 108 110 105 107 106 111 111
```

Es stellt sich nun die Frage, ob es einen Unterschied zwischen der Kieferlänge
von Männchen und Weibchen gibt.

**a) Handelt es sich um gepaarte oder ungepaarte Stichproben? Begründen Sie Ihre
Antwort.**

Ungepaart: Unterschiedliche Anzahl Messungen, es kann nicht jedem $x$ ein eindeutiges $y$ zugewiesen werden.

**b) Formulieren Sie Nullhypothese und Alternativhypothese.**

* $H_0$: Es gibt keinen Unterschied zwischen den Kieferlängen von Männchen und Weibchen: $\mu_x = \mu_y$
* *$H_A$: Es gibt einen Unterschied: $\mu_x \neq \mu_y$

**c) Führen Sie den t-Test nun noch mit Hilfe von Python durch. Geben Sie den resultierenden p-Wert sowie den daraus folgenden Testentscheid an.**

In [38]:
import pandas as pd
import scipy.stats as st
jackals = pd.read_table(r"./dat/jackals.txt", sep = " ")
st.ttest_ind(jackals["M"], jackals["W"], equal_var=False).pvalue

0.0033599524351925016

*Annahme: Signifikanznivieau $\alpha = 0.05$*


Der p-Wert ist 0.0034 und somit kleiner als das Signifikazniveau $\alpha = 0.05$. Die Nullhypothese wird also verworfen.

Es gibt sehr wahrscheinlich einen Unterschied bei der Kieferlänge von männlichen und weiblichen Goldschakalen.

**d) Führen Sie mit Hilfe von Python einen Wilcoxon-Test durch. Geben Sie wiederum p-Wert und Testentscheid an.**

In [40]:
st.mannwhitneyu(jackals["M"], jackals["W"], alternative = "two-sided").pvalue

0.004845462884722891

Der p-Wert ist auch hier (knapp) unter dem Signifikanzniveau, die Nullhypothese wird also verworfen.

**e) Falls die Resultate der beiden Tests unterschiedlich ausgefallen wären, welchem würden Sie eher vertrauen? Weshalb?**

Der Wilcoxon-Test hat in der Regel die grössere Macht. Ausserdem kann der t-Test nur unter Annahme der Normalverteilung verwerden, was hier jedoch nicht zwingend der Fall sein mus.

Aus diesem Gründen ist das Resulat des Wilcoxon-Tests vorzuziehen.

# Aufgabe 8.5

Im Jahr 2013 wurden im Rahmen einer internationalen Zusammenarbeit unter der Leitung der EAWAG in Dübendorf Konzentrationen von illegalen Substanzen im Abwasser von 42 europäischen Städten während einer Woche untersucht (Ort C. et all, *Spatial differences and temporal changes in illicit drug use in Europe quantified by wastewater
analysis*, Addiction 2014 Aug).

Dabei wurden an 7 aufeinanderfolgenden Tagen (6.-12. März) neben anderen Substanzen die medianen Konzentrationen von Ecstasy (MDMA) im Abwasser gemessenen. Aufgrund dieser Studie war eine Aussage einer vielgelesenen Schweizer Gratiszeitung, dass in Zürich viel mehr Drogen konsumiert werden als anderswo.

In der nachfolgenden Tabelle sind für die Städte Zürich und Basel die an den untersuchten Tagen ausgeschiedenen Mengen MDMA aufgeführt. Die Angaben sind in mg pro 1000 Einwohner pro Tag. Nehmen Sie an, dass die täglichen Differenzen $D_i$ zwischen den pro tausend Einwohner ausgeschiedenen Mengen von MDMA im Abwasser von Zürich und Basel unabhängig voneinander normalverteilt mit Erwartungswert
$μ_D$ und Standardabweichung $s_D$ sind.

| Wochentage |  Mi |  Do |  Fr |  Sa | So |  Mo |  Di |
|--- | ---:| ---:| ---:| ---:| ---:| ---:| ---:|
| Zürich     | 16.3|   12.7|   14.0 | 53.3 |117.0 | 62.6| 27.6|
|Basel       |10.4 | 8.91 | 11.7 | 29.9 | 46.3 | 25.0 | 29.4 |

**a) Schätzen Sie aus den Daten den Mittelwert und die Standardabweichung der Differenzen, d.h., $\hat{μ}_D$ und $\hat{\sigma}_D$.**

In [44]:
import numpy as np
from pandas import Series
mdma_zuerich = Series([16.3, 12.7, 14.0, 53.3, 117, 62.6, 27.6])
mdma_basel = Series([10.4, 8.91, 11.7, 29.9, 46.3, 25.0, 29.4])
d = mdma_zuerich - mdma_basel
print("Mean:\t", d.mean())
print("Std:\t", d.std())

Mean:	 20.27
Std:	 26.272304175056032


**b) Handelt es sich um gepaarte oder ungepaarte Stichproben? Begründen Sie Ihre Antwort.**

Ungepaart: es werden zwei unterschiedliche Sachen gemessen, und nicht zweiml das gleiche.

**c) Formulieren Sie die Nullhypothese und die Alternativhypothese, wenn Sie die
Aussage der besagten Gratiszeitung überprüfen möchten.**

* $H_0: \mu_D = \mu_0 = 0$
* $H_A: \mu_D > \mu_0 =  0$

**d) Führen Sie einen statistischen Test mit Hilfe von Python auf dem Signifikanzniveau 95% durch, unter der Annahme, dass die Daten normalverteilt sind.**

**Wie lautet die Teststatistik und wie ist diese unter der Nullhypothese verteilt?**

Teststatistik: $D \sim \mathcal{N}(\mu, \sigma^2)$

In [47]:
st.ttest_ind(mdma_zuerich,mdma_basel, equal_var=False).pvalue

0.2232566584677775

**e) Führen Sie nun einen statistischen Test mit Hilfe von Python auf dem Signifikanzniveau
95% durch, unter der Annahme, dass die Daten nicht normalverteilt
sind.**

In [50]:
st.mannwhitneyu(mdma_zuerich, mdma_basel, alternative='greater').pvalue

0.12507650397969117